jkube-kit/config/service/src/main/java/org/eclipse/jkube/kit/config/service/kubernetes/JibBuildService.java

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2019 Red Hat, Inc.
+ * This program and the accompanying materials are made
+ * available under the terms of the Eclipse Public License 2.0
+ * which is available at:
+ *
+ *     https://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *   Red Hat, Inc. - initial API and implementation
+ */
+package org.eclipse.jkube.kit.config.service.kubernetes;
+
+import com.google.cloud.tools.jib.api.Credential;
+import com.google.cloud.tools.jib.api.JibContainerBuilder;
+import com.google.cloud.tools.jib.api.TarImage;
+import org.eclipse.jkube.kit.build.api.assembly.AssemblyManager;
+import org.eclipse.jkube.kit.build.api.assembly.BuildDirs;
+import org.eclipse.jkube.kit.build.api.auth.AuthConfig;
+import org.eclipse.jkube.kit.build.service.docker.auth.AuthConfigFactory;
+import org.eclipse.jkube.kit.common.KitLogger;
+import org.eclipse.jkube.kit.common.util.EnvUtil;
+import org.eclipse.jkube.kit.config.image.ImageConfiguration;
+import org.eclipse.jkube.kit.config.image.ImageName;
+import org.eclipse.jkube.kit.config.image.RegistryConfig;
+import org.eclipse.jkube.kit.config.image.build.JKubeConfiguration;
+import org.eclipse.jkube.kit.config.service.BuildService;
+import org.eclipse.jkube.kit.config.service.BuildServiceConfig;
+import org.eclipse.jkube.kit.config.service.JKubeServiceException;
+import org.eclipse.jkube.kit.config.service.JKubeServiceHub;
+import org.eclipse.jkube.kit.service.jib.JibServiceUtil;
+
+import java.io.File;
+import java.io.IOException;
+import java.util.Collection;
+import java.util.Objects;
+
+public class JibBuildService implements BuildService {
+    private static final String DOCKER_REGISTRY = "docker.io";
+    private static final String PUSH_REGISTRY = "docker.push.registry";
+    public static final String DOCKER_BUILD_TAR = "docker-build.tar";
```

Comment:
This constant seems related to https://github.com/eclipse/jkube/blob/300fcfdfdc6b40b9820bdcdcae4f59d35870402c/jkube-kit/build/api/src/main/java/org/eclipse/jkube/kit/build/api/assembly/JKubeBuildTarArchiver.java#L55.

Maybe we should share at least the filename in some constant common to both classes (Preferably in the Archiver)